src/com/etendoerp/etendorx/services/wrapper/EtendoResponseWrapper.java

```diff
@@ -0,0 +1,47 @@
+package com.etendoerp.etendorx.services.wrapper;
+
+import javax.servlet.ServletOutputStream;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpServletResponseWrapper;
+import java.io.CharArrayWriter;
+import java.io.IOException;
+import java.io.PrintWriter;
+
+public class EtendoResponseWrapper extends HttpServletResponseWrapper {
+  private CharArrayWriter charArrayWriter = new CharArrayWriter();
+  private PrintWriter writer = new PrintWriter(charArrayWriter);
+
+  public EtendoResponseWrapper(HttpServletResponse response) {
+    super(response);
+  }
+
+  @Override
+  public PrintWriter getWriter() throws IOException {
+    return writer;
+  }
+
+  @Override
+  public ServletOutputStream getOutputStream() throws IOException {
+    throw new UnsupportedOperationException("This wrapper only supports getWriter().");
```

Comment:
Error must be ad message

Reply:
This is a message between systems, is preferable to keep in english as a common language